### DISCUSSION TITLE
---

### DISCUSSION THREAD

**Discussion by cyberrrr** [Rapptz/discord.py#6346] [Q&A]
---

**Comment by SebbyLaw** [comment]
Discord does not dispatch the full member list to users anymore. Use a bot account with server members intents enabled. Also see #5782

**Reply by SebbyLaw** [reply]
Your client makes requests to discord that discord.py doesn't. Use a regular bot account for methods to properly work as intended.

Selfbots are against terms of service and discord.py will not support them in the future.

**Reply by SebbyLaw** [reply]
If you want to know how the client does it, feel free to reverse engineer it. You should be using a bot account for this.

**Comment by JavascriptLearner815** [comment]
Self-bot mode seems against the TOS, but to access the full members list, you MUST now enable "SERVER MEMBERS INTENT" on your bot dashboard, even though you have "access to everything the user does". And, `intents.members = True` must be set as well.

BASICALLY, "YOU'VE SETUP THE INTENTS" IS INCORRECT.
